client/src/components/css/LandingPageNews.css

```diff
@@ -0,0 +1,41 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto);
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+.headingTextNews {
+    font-family: Roboto;
+    font-size: 36px;
+    font-weight: bold;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: #4a4a4a;
+    text-align: left;
+}
+
+.backgroundNews {
+    width: 100%;
+    height: 50%;
+    padding: 3%;
+}
+
+.moreNews {
```

Comment:
Add the cursor property here so that when you hover over "More news" the mouse icon changes so the user knows that they can click on it. It's the same way for a few other elements on the landing page, I'll open a PR to change it

Reply:
Fixed